go/hack/compat.go

```diff
@@ -0,0 +1,36 @@
+//go:build !gc || wasm
+
+package hack
+
+func String(b []byte) (s string) {
```

Comment:
Similarly - what about comments for these public methods?

Reply:
Fixed!